src/csl/rendering/names.rs

```diff
@@ -597,7 +597,7 @@ fn add_names<T: EntryLike>(
                 persons.len() - 1,
             );
         }
-    } else if has_et_al {
+    } else if ctx.instance.sorting.is_none() && has_et_al {
```

Comment:
There are other checks for `has_et_al` above. Are we sure we only have to change this step? 

I suppose that there could be a bit of a debate on whether changing the sort behavior too much would be welcome as well. In principle, it seems like the user would find it unexpected for sort to not sort based on the final text, which includes et al. But at least your argument seems reasonable.

Overall, maybe my question is how much do we want to modify the sort to ignore et al.